src/middleware/fhir/4_0_0/controllers/generic_controller.js

```diff
@@ -0,0 +1,212 @@
+const handler = require('../../fhir-response-util');
```

Comment:
Copy and change this code from the node-fhir-server-core so it can support streaming